lifeisyoung.js.org

lifeisyoung opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```lifeisyoung.js.org``` to make it work.
I'll merge after we have "contact".

lifeisyoung commented:
> 
> 
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `lifeisyoung.js.org` to make it work.
> I'll merge after we have "contact".



> 
> 
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `lifeisyoung.js.org` to make it work.
> I'll merge after we have "contact".

i have already added the CNAME file, i am so sorry!

indus commented:
Welcome https://lifeisyoung.js.org
